backend/tests/Squidex.Domain.Apps.Entities.Tests/Contents/DomainObject/Guards/GuardContentTests.cs

```diff
@@ -28,7 +29,9 @@ public class GuardContentTests : IClassFixture<TranslationsFixture>
         private readonly IContentRepository contentRepository = A.Fake<IContentRepository>();
         private readonly NamedId<DomainId> appId = NamedId.Of(DomainId.NewGuid(), "my-app");
         private readonly ClaimsPrincipal user = Mocks.FrontendUser();
+        private readonly ClaimsPrincipal userWithDeletePermission = Mocks.FrontendUser(permission: Permissions.AppContentsDelete);
```

Comment:
I don't see why we need this, we should use a general permission here. If we need this only for one method we should not make it static.